CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 2.2.2
+  - Now depend on `logstash-core-plugin-api` instead of `logstash-core`, this remove the need to mass update 
```

Comment:
maybe:

Depend on `logstash-core-plugin-api` instead of `logstash-core`, removing the need to mass update plugins on major releases of logstash